l10n_br_pos/models/l10n_br_account_product.py

```diff
@@ -11,6 +20,30 @@ class L10nbrAccountCFOP(models.Model):
 
     is_pos = fields.Boolean(
         string=u"Permitida no POS",
-        help=u"Marque esta selaçao para que a cfop possa ser utililizada no "\
-             u"NFC-E/SAT"
-    )
+        help=u"Marque esta selaçao para que a cfop possa ser utililizada no "
+             u"NFC-E/SAT")
+
+
+class L10nbrAccountDocumentRelated(models.Model):
+    _inherit = 'l10n_br_account_product.document.related'
+
+    pos_order_related_id = fields.Many2one('pos.order',
+                                           string=u'Documento Fiscal SAT',
+                                           ondelete='cascade', select=True)
+
+    document_type = fields.Selection(
+        [('nf', 'NF'), ('nfe', 'NF-e'), ('cte', 'CT-e'),
+         ('nfrural', 'NF Produtor'), ('cf', 'Cupom Fiscal'),
+         ('sat', 'CF-e/SAT')],
+        'Tipo Documento', required=True)
+
+    @api.onchange('pos_order_related_id')
```

Comment:
Será que não deveríamos ter também um `onchange` no campo `document_type` para apagar o `pos_order_related_id` se o `document_type` for mudado para algo que não é `sat`?